2024-07-06 09:14:55,480 - root - logging_functions.py - ERROR - <module> - Testing saving logs to file.
